vm/src/objtype.rs

```diff
@@ -170,6 +179,14 @@ pub fn new(typ: PyObjectRef, name: &str, bases: Vec<PyObjectRef>, dict: PyObject
     ))
 }
 
+fn type_str(vm: &mut VirtualMachine, args: PyFuncArgs) -> PyResult {
+    // TODO: fix macro:
+    // arg_check!(vm, args, required = [(obj, Some(vm.ctx.type_type()))]);
```

Comment:
What needs fixing here?

Reply:
The macro contains objtype, but we are in this place already in objtype, so it cannot find objtype::

Reply:
Wow, I read my own response back... does that make sense?